test/run_test.py

```diff
@@ -13,45 +13,61 @@
     subprocess.run("tar xf example_set.tar.bz2", shell=True, check=True)
 
 #easy run
-sys.stderr.write("Running database creation + DBSCAN model fit + fit refinement (--easy-run)\n")
-subprocess.run("python ../poppunk-runner.py --easy-run --r-files references.txt --min-k 13 --k-step 3 --output example_db --full-db --qc-filter prune --overwrite", shell=True, check=True)
+sys.stderr.write("Running database creation (--create-db)\n")
+subprocess.run("python ../poppunk-runner.py --create-db --r-files references.txt --min-k 13 --k-step 3 --output example_db --qc-filter prune --overwrite", shell=True, check=True)
 
 # create database with different QC options
 sys.stderr.write("Running database QC test (--create-db)\n")
-subprocess.run("python ../poppunk-runner.py --create-db --r-files references.txt --min-k 13 --k-step 3 --output example_qc --full-db --qc-filter continue --length-range 2000000 3000000 --overwrite", shell=True, check=True)
+subprocess.run("python ../poppunk-runner.py --create-db --r-files references.txt --min-k 13 --k-step 3 --output example_qc --qc-filter continue --length-range 2000000 3000000 --overwrite", shell=True, check=True)
 
 #fit GMM
-sys.stderr.write("Running GMM model fit (--fit-model)\n")
-subprocess.run("python ../poppunk-runner.py --fit-model --distances example_db/example_db.dists --ref-db example_db --output example_db --full-db --K 4 --microreact --cytoscape --overwrite", shell=True, check=True)
+sys.stderr.write("Running GMM model fit (--fit-model gmm)\n")
+subprocess.run("python ../poppunk-runner.py --fit-model bgmm --distances example_db/example_db.dists --ref-db example_db --output example_db --K 4 --overwrite", shell=True, check=True)
+
+#fit GMM
+sys.stderr.write("Running DBSCAN model fit (--fit-model dbscan)\n")
+subprocess.run("python ../poppunk-runner.py --fit-model dbscan --distances example_db/example_db.dists --ref-db example_db --output example_dbscan --overwrite", shell=True, check=True)
 
 #refine model with GMM
-sys.stderr.write("Running model refinement (--refine-model)\n")
-subprocess.run("python ../poppunk-runner.py --refine-model --distances example_db/example_db.dists --ref-db example_db --output example_refine --neg-shift 0.8 --overwrite", shell=True, check=True)
+sys.stderr.write("Running model refinement (--fit-model refine)\n")
+subprocess.run("python ../poppunk-runner.py --fit-model refine --distances example_db/example_db.dists --ref-db example_db --output example_refine --neg-shift 0.8 --overwrite", shell=True, check=True)
```

Comment:
Regarding making the code more user-friendly - in the absence of flags, could we assume that `--distances` and `--output` could be inferred from `--ref-db` for `--fit-model` commands? It would make the typical commands look much easier to run.

Reply:
Yeah, very nice idea, I will see if there's something sensible to be done there